android/modules/platform/src/java/ti/modules/titanium/platform/DisplayCapsProxy.java

```diff
@@ -67,10 +67,16 @@ public String getDensity() {
 				return "high";
 			case DisplayMetrics.DENSITY_MEDIUM :
 				return "medium";
-			case 320 : // DisplayMetrics.DENSITY_XHIGH (API 9)
+			case DisplayMetrics.DENSITY_XHIGH :
```

Comment:
Are you sure we can use this constant? The original code said something about this only being available in API 9 and higher? Or is it now available in all API's we support?

If we can use the constants, why not implement all of them (http://developer.android.com/reference/android/util/DisplayMetrics.html) and map in-betweens like `DENSITY_560` to `560` and `DENSITY_TV` to `tv` to stay in line with the original platform values?